skimage/transform/tests/test_geometric.py

```diff
@@ -245,11 +246,21 @@ def test_fundamental_matrix_residuals():
     assert_almost_equal(tform.residuals(src, dst)**2, [0, 0.5, 2])
 
 
-def test_fundamental_matrix_forward():
+@pytest.mark.parametrize('array_like_input', [False, True])
```

Comment:
A bit confusing for me at first read, because an array is *a fortiori* array-like... :upside_down_face: 
But I admit that `not_array_but_array_like_input` would be too long!

Reply:
@mkcor I'd be in the `not_array_but_array_like_input` team! :joy: It wouldn't bite us in the back in the future.
What do you think, @grlee77?